scripts/linters/general_purpose_linter.py

```diff
@@ -704,12 +733,13 @@ def _check_bad_patterns(self):
         summary_messages = []
         all_filepaths = [
             filepath for filepath in self.all_filepaths if not (
-                filepath.endswith('general_purpose_linter.py'))]
+                filepath.endswith('general_purpose_linter.py') or (
+                    filepath.endswith('general_purpose_linter_test.py')))]
```

Comment:
Why is this file excluded?

Reply:
Because this file contains the regexes for bad patterns, if we run those checks on this files, then we get too many positives.